dependencies.yaml

```diff
@@ -57,12 +55,6 @@ dependencies:
     - path: scripts/versions
       match: buildah
 
-  - name: runc
```

Comment:
I would retain this logic (similar to CI). There may be instances where we need to stick to a specific version due to regressions in runc

Reply:
Alright, reverted that change.